contracts/src/core/EigenDAServiceManager.sol

```diff
@@ -182,8 +188,68 @@ contract EigenDAServiceManager is Initializable, OwnableUpgradeable, EigenDAServ
         return uint32(block.number) + STORE_DURATION_BLOCKS + BLOCK_STALE_MEASURE;
     }
 
-    /// @dev need to override function here since its defined in both these contracts
-    function owner() public view override(OwnableUpgradeable, IServiceManager) returns (address) {
-        return OwnableUpgradeable.owner();
+    /**
+     * @notice Returns the list of strategies that the operator has potentially restaked on the AVS
+     * @param operator The address of the operator to get restaked strategies for
+     * @dev This function is intended to be called off-chain
+     * @dev No guarantee is made on whether the operator has shares for a strategy in a quorum or uniqueness 
+     *      of each element in the returned array. The off-chain service should do that validation separately
+     */
+    function getOperatorRestakedStrategies(address operator) external view returns (address[] memory) {
+        bytes32 operatorId = registryCoordinator.getOperatorId(operator);
+        uint256 quorumBitmap = registryCoordinator.getCurrentQuorumBitmap(operatorId);
+        bytes memory quorumBytesArray = BitmapUtils.bitmapToBytesArray(quorumBitmap);
+
+        uint256 strategiesLength;
+        for (uint i = 0; i < quorumBytesArray.length; i++) {
+            uint8 quorumNumber = uint8(quorumBytesArray[i]);
+            strategiesLength += stakeRegistry.strategyParamsLength(quorumNumber);
+        }
+
+        address[] memory restakedStrategies = new address[](strategiesLength);
+        uint256 index;
+        for (uint i = 0; i < quorumBytesArray.length; i++) {
+            uint8 quorumNumber = uint8(quorumBytesArray[i]);
+            uint256 strategyParamsLength = stakeRegistry.strategyParamsLength(quorumNumber);
+            for (uint j = 0; j < strategyParamsLength; j++) {
+                IStakeRegistry.StrategyParams memory strategyParams = stakeRegistry.strategyParamsByIndex(quorumNumber, j);
+                restakedStrategies[index] = address(strategyParams.strategy);
+                ++index;
+            }
+        }
+
+        return restakedStrategies;
+    }
+
+    /**
+     * @notice Returns the list of strategies that the AVS supports for restaking
+     * @dev This function is intended to be called off-chain
+     * @dev No guarantee is made on uniqueness of each element in the returned array. 
+     *      The off-chain service should do that validabution separately
+     */
+    function getRestakeableStrategies() external view returns (address[] memory) {
+        uint256 quorumBitmap;
+        uint256 strategiesLength;
+        for (uint8 i = 0; i < type(uint8).max; i++) {
```

Comment:
there's a lot a duplication between these functions. we should probably just have

`getRestakeableStrategies(uint256 quorumBitmap)` and reuse that